backend/src/test/scala/com/softwaremill/bootzooka/user/UserApiTest.scala

```diff
@@ -285,7 +325,19 @@ class UserApiTest extends BaseTest with Eventually with TestDependencies {
 
     // then
     response1.shouldDeserializeTo[UpdateUser_OUT]
-    getUser(apiKey).shouldDeserializeTo[GetUser_OUT].login shouldBe newLogin
-    getUser(apiKey).shouldDeserializeTo[GetUser_OUT].email shouldBe newEmail
+    getUser(apiKey)
+      .map(_.body.shouldDeserializeTo[GetUser_OUT])
+      .map(r => {
+        r.login shouldBe newLogin
+        r.email shouldBe newEmail
+      })
+      .unsafeRunSync()
+  }
+
+  def getUser(apiKey: String): IO[client3.Response[Either[String, String]]] = {
+    basicRequest
+      .get(uri"http://localhost:8080/api/v1/user")
+      .header("Authorization", s"Bearer $apiKey")
+      .send(backendStub)
```

Comment:
I think we can use `.unwrap` here just as in the original to avoid using `IO` in the tests